cdci_data_analysis/flask_app/templates/submitted.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<body>
+
+Dear user,
+
+you receive this message because you submitted a request to get from the service provided by University of Geneva available at the URL {}.
```

Comment:
"University of Geneva" should be variable (though default)

Reply:
You mean that will change depending on the deployment? Makes sense.